tsl/test/sql/CMakeLists.txt

```diff
@@ -73,6 +84,34 @@ set(TEST_FILES_DEBUG
     tsl_tables.sql
     license.sql)
 
+endif(CMAKE_BUILD_TYPE MATCHES Debug)
+
+set(SOLO_TESTS_postgresql
+    bgw_reorder_drop_chunks
+    compress_bgw_reorder_drop_chunks
+    compression_ddl
+    cagg_bgw
+    cagg_ddl
+    cagg_ddl_dist_ht
+    cagg_dump
+    data_fetcher
+    dist_util
+    move
+    remote_connection_cache
+    remote_copy
+    remote_txn
+    remote_txn_resolve
+    reorder
+    telemetry_distributed)
+
+# Test files for the max_workers_8 configuration that configures TimescaleDB
+# with a max of 8 workers.
```

Comment:
Nope, it's actually 16.

Reply:
It is actually 8 background workers, but 16 workers in total. I realize the description is not clear, so will fix it.

Reply:
Updated.